Makefile

```diff
@@ -115,3 +115,21 @@ report-celery-beat:
 
 admin-user:
 	superset fab create-admin
+
+# Create a postgres instance using docker
+# CLI: $ psql postgresql://postgres:1234@localhost:5432/postgres
+# superset_config.py: SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:1234@localhost:5432/postgres'
+db:
+	# start postgres with detached mode
+	docker run -d -p 127.0.0.1:5432:5432 -e POSTGRES_PASSWORD="1234" --name pg postgres:alpine;\
```

Comment:
Any plaintext passwords should be avoided as much as possible, at least, providing an approach that the password pass from a command parameter.

Reply:
the makefile
```Makefile
db:
	docker run -d -p 127.0.0.1:5432:5432 -e POSTGRES_PASSWORD=$(PWD) --name pg postgres:alpine;
```

to invoke:
```shell
PWD=1234 make db
```

Reply:
> even if this is for development? I'm trying to make this super easy for people to spin up superset with one command, but if you really think we should have a way to configure the password let me know

No, If you want to use it yourself, you can do it by yourself and local. If you publish it in a public codebase, please keep the basic security rules.